### PR TITLE
add nova network

### PR DESCRIPTION
**Pull request by nova-network-inc**
Additions # .

Changes proposed in this pull request:
- Add Nova Network

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -413,6 +413,20 @@
     "start": 4992871,
     "logo": "ipfs://QmSZvT9w9eUDvV1YVaq3BKKEbubtNVqu1Rin44FuN4wz11"
   },
+  "87": {
+    "key": "87",
+    "name": "Nova Network",
+    "shortName": "Nova",
+    "chainId": 87,
+    "network": "mainnet",
+    "multicall": "0x7A5a7579eb8DdEd352848cFDD0a5530C4e56FF7f",
```
Comment (ChaituVR) [inline src/networks.json:422]: Is there a way to verify this contract?

File: src/networks.json [hunk 2]
```diff
@@ -413,6 +413,20 @@
...
+    "rpc": [
+      "https://dev.rpc.novanetwork.io/"
+    ],
+    "explorer": "https://explorer.novanetwork.io/",
+    "start": 1000000,
```
Comment (ChaituVR) [inline src/networks.json:427]: should be block number at multicall contract creation

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 2 above)
Comment (nova-network-inc) [reply src/networks.json]: Deployed a new contract and changed the block number.

File: src/networks.json (hunk 1 above)
Comment (nova-network-inc) [reply src/networks.json]: You can have a look here https://github.com/nova-network-inc/fusion-public-contracts/blob/main/NovaMulticall.sol the explorer isn't indexed, so doesn't support verification just yet. Working through that.

**Comment by nova-network-inc** [comment]
All concerns addressed. Please let me know if there's anything else you might need 😃

**Comment by ChaituVR** [comment]
Looks good, except that we need to verify multicall contract through explorer

**Comment by nova-network-inc** [comment]
> Looks good, except that we need to verify multicall contract through explorer

Okay, https://novascan.io/ is indexed and verification via CLI, I'll do it and send the link through later on today.

**Comment by nova-network-inc** [comment]
> Looks good, except that we need to verify multicall contract through explorer

I had to redeploy it to verify, and the explorer has a UI glitch which prevents it to load the artifact, but you will see the contract verified and all the calls in there.

https://novascan.io/address/0x07f6b55a5dae24ac8011cc2cb815de8316e2b965?tab=contract

Let me know if anything else is needed.

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Hey @nova-network-inc 

Nova network's node returns this error when used on our server, 
```
Error: unable to verify the first certificate
```
but works fine from any front end, any idea?

**Comment by nova-network-inc** [comment]
> Hey @nova-network-inc
> 
> Nova network's node returns this error when used on our server,
> 
> ```
> Error: unable to verify the first certificate
> ```
> 
> but works fine from any front end, any idea?

It was a problem with the RPC's proxy. It has been fixed now. Thanks @ChaituVR

**Pull request merged**